### PR TITLE
Added Elasticsearch Article

### PR DESCRIPTION
**Pull request by BenjaminBruenau**

### CODE REVIEW

File: docs/presentations/articles/elasticsearch.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+Elasticsearch is often mentioned as part of the Elastic or ELK Stack.
+The ELK Stack, consisting of Elasticsearch, Logstash, and Kibana, plays a crucial role in handling data. Elasticsearch is
+responsible for the search and analytics part, Logstash for data processing and forwarding, and Kibana for visualizing
+and managing stored data. Other services like Beats are often integrated for various functionalities, e.g. collection of data.
+
+!!! example "Elastic Stack"
+    
+    Logstash is usually used to ingest data from various sources into Elasticsearch (optionally parsing it beforehand).
+    Beats are Agents, attached to for example Applications to collect logs or other metrics. Kibana utilizes the powerful search engine of Elasticsearch to then visualize the data
```
Comment (pkeilbach) [inline docs/presentations/articles/elasticsearch.md:35]: missing dot

```suggestion
    Beats are Agents, attached to for example Applications to collect logs or other metrics. Kibana utilizes the powerful search engine of Elasticsearch to then visualize the data.
```

File: docs/presentations/articles/assets/elasticsearch-index-example.png
Comment (pkeilbach) [inline docs/presentations/articles/assets/elasticsearch-index-example.png:1]: Could you change those to English examples? "Andi" is fine though 😄

**Review by pkeilbach — CHANGES_REQUESTED** [review]
Great job, @BenjaminBruenau! Just small comments, then we are good to go!

File: docs/presentations/articles/assets/elasticsearch-index-example.png
Comment (BenjaminBruenau) [reply docs/presentations/articles/assets/elasticsearch-index-example.png:1]: I also added another entry to the forward index example for the sake of completion

**Review by pkeilbach — APPROVED** [review]
LGTM! 🚀

**Pull request merged**